docs/reference/indices/resolve-cluster.asciidoc

```diff
@@ -263,7 +297,12 @@ GET /_resolve/cluster/not-present,clust*:my-index*,oldcluster:*?ignore_unavailab
     "connected": false,           <3>
     "skip_unavailable": false
   },
-  "oldcluster": {         <4>
+  "cluster_three": {
+    "connected": false,
+    "skip_unavailable": false,
+    "error": "Request timed out before receiving a response from the remote cluster"  <4>
```

Comment:
Not for this patch, but I wonder if it'd help if the error message included the actual timeout value. So the user would know whether the cluster failed to respond in 1s (maybe the timeout too low?) or in 300s (definitely it's dead and also maybe timeout too long).